cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -859,7 +934,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                     tabs[1].editors = [StaffLockEditor];
                 } else if (xblockInfo.isSequential()) {
                     tabs[0].editors = [ReleaseDateEditor, GradingEditor, DueDateEditor];
-                    tabs[1].editors = [ContentVisibilityEditor];
+                    tabs[1].editors = [ContentVisibilityEditor, ShowCorrectnessEditor];
```

Comment:
@pomegranited I touched base with @roderickmorales about this off-line, and given that the other pre-existing sections will also need to have their font sizes adjusted, we decided that you don't need to take on this change as part of this PR.

However, if you WANT to, the various issues are documented in https://openedx.atlassian.net/browse/TNL-6824.